### PR TITLE
feat(BAC-49): added endpoint that returns media files uploaded by the particular user

### PR DESCRIPTION
**Pull request by babafemiolatona**
METHOD: GET

Added endpoint that returns recent media files uploaded by the particular user

 args:
        org_id (str): A unique identifier of the organization.
        member_id (str): A unique identifier of the member.

 returns:
        A dictionary containing a list of media objects.

[Linear link]( https://linear.app/team-socket/issue/BAC-49/create-endpoint-for-viewing-all-recent-files-uploaded-by-the-current)

### CODE REVIEW

**Comment by bolexs** [comment]
> METHOD: GET
> 
> This endpoint returns recent media files uploaded by that user
> 
> Linear link: https://linear.app/team-socket/issue/BAC-49/create-endpoint-for-viewing-all-recent-files-uploaded-by-the-current

Please let you pr be more explanatory

File: backend/requirements.txt [hunk 1]
```diff
@@ -1,64 +1,20 @@
...
-typing-extensions==3.10.0.2
-urllib3==1.26.7
-uvicorn[standard]
-virtualenv==20.9.0
-watchgod==0.7
-websockets==10.0
-wrapt==1.13.2
+starlette==0.21.0
+tomli==2.0.1
+typing_extensions==4.4.0
```
Comment (FlorenceEgwu) [inline backend/requirements.txt]: Is the a genuine reason why you decided to tamper with the requirements file?

Comment (babafemiolatona) [reply backend/requirements.txt]: I was having some conflicts while trying to install dependencies

Comment (zxenonx) [reply backend/requirements.txt]: Please, what happened here?
Some essential packages have been removed... pytest, bson, etc. Why?
Please kindly fix that?

**Review by zxenonx — CHANGES_REQUESTED** [review]
Thanks. 
I left a comment.
@babafemiolatona

File: backend/requirements.txt (hunk 1 above)
Comment (babafemiolatona) [reply backend/requirements.txt]: Okay. I'll fix it now

**Pull request closed without merging**